include/barcodes/datamatrix.php

```diff
@@ -230,38 +237,62 @@ class Datamatrix {
 	 * This is the class constructor.
 	 * Creates a datamatrix object
 	 * @param string $code Code to represent using Datamatrix.
+	 * @param string $shape Shape of datamatrix code (R = rectangular, S = square)
```

Comment:
```suggestion
	 * @param string $shape Shape of datamatrix code (R = rectangular, S = square)
	 * @phpstan-param 'R'|'S' $shape
```

Reply:
@williamdes Just saw your comment recommendations. Unfortunately only after I added the latest change -> Update will follow...
I've never used phpstan. Is the last line there to specify the parameters for automated testing?

Reply:
You should check out phpstan, it is for static analysis

Here is an example of what my proposal would do https://phpstan.org/r/f04d0c5e-0a40-4959-b03a-fdee4430a29a

Reply:
If I understand correctly, this does not generate an automatic verification script.
As you may have seen, I have extended the parameter so that you can specify the desired dimensions (e.g. 48x48). Can this also be specified in a kind of wildcard form, or would each valid combination have to be specified separately?

Reply:
> If I understand correctly, this does not generate an automatic verification script.

This is used by automatic tools, in this repo:
https://github.com/tecnickcom/TCPDF/blob/5fce932fcee4371865314ab7f6c0d85423c5c7ce/.github/workflows/tests.yml#L121

But for people that will integrate TCPDF, it will also throw warnings if they use phpstan and pass something else than the values you did set